js/mexc3.js

```diff
@@ -0,0 +1,3659 @@
+            //       [
+            //         1640804880000,
+            //         "47482.36",
+            //         "47482.36",
+            //         "47416.57",
+            //         "47436.1",
+            //         "3.550717",
+            //         1640804940000,
+            //         "168387.3"
+            //       ],
+            //     ]
+            //
+            candles = response;
+        } else if (market['swap']) {
+            if (since !== undefined) {
+                request['start'] = parseInt (since / 1000);
+            }
+            const priceType = this.safeString (params, 'price', 'default');
+            params = this.omit (params, 'price');
+            const method = this.getSupportedMapping (priceType, {
+                'default': 'contractPublicGetKlineSymbol',
+                'index': 'contractPublicGetKlineIndexPriceSymbol',
+                'mark': 'contractPublicGetKlineFairPriceSymbol',
+            });
+            const response = await this[method] (this.extend (request, params));
+            //
+            //     {
+            //         "success":true,
+            //         "code":0,
+            //         "data":{
+            //             "time":[1634052300,1634052360,1634052420],
+            //             "open":[3492.2,3491.3,3495.65],
+            //             "close":[3491.3,3495.65,3495.2],
+            //             "high":[3495.85,3496.55,3499.4],
+            //             "low":[3491.15,3490.9,3494.2],
+            //             "vol":[1740.0,351.0,314.0],
+            //             "amount":[60793.623,12260.4885,10983.1375],
+            //         }
+            //     }
+            //
+            const data = this.safeValue (response, 'data');
+            candles = this.convertTradingViewToOHLCV (data, 'time', 'open', 'high', 'low', 'close', 'vol');
+        }
+        return this.parseOHLCVs (candles, market, timeframe, since, limit);
+    }
+
+    async fetchIndexOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        return await this.fetchOHLCV (symbol, timeframe, since, limit, this.extend ({ 'price': 'index' }, params));
+    }
+
+    async fetchMarkOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        return await this.fetchOHLCV (symbol, timeframe, since, limit, this.extend ({ 'price': 'mark' }, params));
+    }
+
+    parseOHLCV (ohlcv, market = undefined) {
+        return [
+            this.safeInteger (ohlcv, 0),
+            this.safeNumber (ohlcv, 1),
+            this.safeNumber (ohlcv, 2),
+            this.safeNumber (ohlcv, 3),
+            this.safeNumber (ohlcv, 4),
+            this.safeNumber (ohlcv, 5),
+        ];
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (symbols !== undefined) {
+            request['symbol'] = symbols.join (',');
```

Comment:
You're using here symbols directly instead of the marketIds 

Example:
```
const marketIds = this.marketIds (symbols);
 request['symbol'] = marketIds.join (',');
```

Reply:
good catch! 
swaps needed additional change too. tnx

Reply:
we have simplified method for it  `this.marketIds (symbols)`.